src/Consolonia.Core/Drawing/ConsoloniaPlatformRenderInterfaceContext.cs

```diff
@@ -20,6 +21,13 @@ public IRenderTarget CreateRenderTarget(IEnumerable<object> surfaces)
             return new RenderTarget(surfaces);
         }
 
+        public IDrawingContextLayerImpl CreateOffscreenRenderTarget(PixelSize pixelSize, double scaling)
+        {
+            throw new NotImplementedException();
+        }
+
         public bool IsLost => false;
+
+        public IReadOnlyDictionary<Type, object> PublicFeatures => new Dictionary<Type, object>();
```

Comment:
May be lets replace with just {get;} = new Dictionary<...>();  ? Just not to create an object on each call?

Reply:
This is a shorthand for that:
  private string _name;  // the name field
  public string Name => _name;     // the Name property with get only accessor

Reply:
I mean
```
public IReadOnlyDictionary<Type, object> PublicFeatures => new Dictionary<Type, object>();
```
is shorthand for
```
public IReadOnlyDictionary<Type, object> PublicFeatures
        {
            get
            {
                return new Dictionary<Type, object>();
            }
        }
```
New dictionary is initialized on each property call.

While 
```
public IReadOnlyDictionary<Type, object> PublicFeatures { get; } = new Dictionary<Type, object>();
```
is shorthand for
```
private readonly IReadOnlyDictionary<Type, object> _publicFeatures = new Dictionary<Type, object>();
        public IReadOnlyDictionary<Type, object> PublicFeatures => _publicFeatures;
```
where only one dictionary is initialized.

Reply:
got it, dang, that's subtle and dangerous. I'm going to have be a lot more careful using that pattern.

Reply:
Interesting thing that CodeRabbit did not notice this.